packages/react-reconciler/src/__tests__/useMutableSourceHydration-test.js

```diff
@@ -380,79 +380,4 @@ describe('useMutableSourceHydration', () => {
         'an issue.',
     ]);
   });
-
-  // @gate !enableSyncDefaultUpdates
-  // @gate enableUseMutableSource
-  it('should detect a tear during a higher priority interruption', () => {
```

Comment:
I deleted this test instead of rewriting it since useMutableSource is slated for removal anyway